version/version.go

```diff
@@ -1,6 +1,6 @@
 package version
 
 var (
-	Version            = "2.2.0"
-	IntegreatlyVersion = "2.2.0"
+	Version            = "2.2.1"
+	IntegreatlyVersion = "2.2.1"
```

Comment:
This is usually done by Delorean.
I am guessing that you didn't mean to commit this change.

Reply:
I did not :-)